lib/postprocessing/image.js

```diff
@@ -212,7 +259,9 @@ async function applyOperations(img, instructions) {
         img.resize(instructions.width || null, instructions.height || null, ">");
     }
 
-    // TODO: inject watermarking
+    if (typeof instructions.watermark === 'object'){
```

Comment:
`{}` is an empty object. Would it crash the `applyWatermark` call?

Reply:
no. it checks for the required fields (`image`): https://github.com/adobe/asset-compute-sdk/pull/58/files/6d1f1a6fab3d0239886fcbc99a81894679942d39#diff-c8cfb029f1e204a8f6da461ffa1464b0R224-R226